solidity/contracts/bridge/Bridge.sol

```diff
@@ -32,12 +51,23 @@ pragma solidity 0.8.4;
 ///         about the sweep increasing appropriate balances in the Bank.
 /// @dev Bridge is an upgradeable component of the Bank.
 contract Bridge {
+    using BTCUtils for uint256;
+    using BTCUtils for bytes;
+    using BytesLib for bytes;
+    using ValidateSPV for bytes;
+    using ValidateSPV for bytes32;
+
     struct DepositInfo {
         uint64 amount;
         address vault;
         uint32 revealedAt;
     }
 
+    // confirmations on the Bitcoin chain
+    uint256 public constant TX_PROOF_DIFFICULTY_FACTOR = 6;
+    // TODO: Make it updatable
+    IRelay public immutable relay;
```

Comment:
Can we add a `@notice` document that field?

Reply:
Done.